### PR TITLE
Jpk reactify

### PR DESCRIPTION
**Pull request by jpknoll**

### CODE REVIEW

**Comment by jcubbage** [comment]
When I run this and go to http://localhost:58369/home/react, I hit an error in vendor.js, line 31596, could not find module "react-router-dom". When I goggled, looksed like it was a dependency that react-router forgot to put in their dependencies, so npm doesn't download. Ticket I read said it was fixed, but maybe not in the version we're using. Should react-router-dom be added to the package.json? Or maybe update react-router?

**Comment by srkirkland** [comment]
@jcubbage Well we aren't using react-router anyway so maybe @jpknoll can fix that

File: Keas.Mvc/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "sloth-web",
```
Comment (jSylvestre) [inline Keas.Mvc/package.json]: @jpknoll Sloth/Keas...

**Comment by jpknoll** [comment]
@jcubbage Give it another go. I need to remove one last reference in the webpack configuration.

**Comment by jcubbage** [comment]
Now getting JS error at line 79, column 154 in main.js. Just says syntax error. That line is the Hot Module Replacement line, so possible something else isn't installed yet? Not sure it if this is relevant, but I'm running without local admin. Had to temporarily grant admin to get node.js installed and npm happy.

**Comment by jcubbage** [comment]
Windows 8 sucks apparently.

**Pull request merged**